gradle/libs.versions.toml

```diff
@@ -0,0 +1,43 @@
+[versions]
+assertj = "3.24.2"
```

Comment:
Couple versions here are outdated.
Same for the Gradle version.

Reply:
Already renovate bot is configured to handle that and gradle version update can be done in follow-up? Because by the time PR gets merged new releases can be out!

Reply:
updated